docs/docsite/rst/dev_guide/developing_modules_in_bulk.rst

```diff
@@ -101,7 +100,7 @@ After publishing your PR on https://github.com/ansible/ansible a Shippable CI te
 If you need further advice join the `#ansible-devel` IRC channel (details in "Where to get support")
 
 
-Please don't force push your PRs, we prefer individual commits in the same PR, the PR will be squashed and merged by the Ansible Core Team when it's ready to be accepted. This is required to allow reviews to see what's changed since they started looking at your PR.
+Please don't "squash" your PRs, we prefer individual commits in the same PR, the PR will be squashed and merged by the Ansible Core Team when it's ready to be accepted. This is required to allow reviews to see what's changed since they started looking at your PR.
```

Comment:
again, i prefer squashing on the user side, it is easier for me to rereview the whole thing